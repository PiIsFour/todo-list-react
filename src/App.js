import React from 'react'
import './App.css'

import ListsOverview from './components/ListsOverview'
import TodoList from './components/TodoList'

function App() {
	return (
		<div className="App">
			<h1 className="App-header">Stuff I need to do</h1>
			<ListsOverview />
			<TodoList />
		</div>
	)
}

export default App
