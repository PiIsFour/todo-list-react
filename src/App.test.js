import React from 'react'
import { shallow } from 'enzyme'

import App from './App'

describe('App', () => {
	it('snapshot matches', () => {
		const wrapper = shallow(<App />)
		expect(wrapper).toMatchSnapshot()
	})

	it('renders header', () => {
		const wrapper = shallow(<App />)
		expect(wrapper.find('.App-header')).toHaveText('Stuff I need to do')
	})
})
