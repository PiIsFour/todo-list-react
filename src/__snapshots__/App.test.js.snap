// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot matches 1`] = `
<div
  className="App"
>
  <h1
    className="App-header"
  >
    Stuff I need to do
  </h1>
</div>
`;
